src/middleware/jsonSchema.js

```diff
@@ -1027,8 +1025,1012 @@ const cveSchema5 = {
+}
+
+const rejectSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: 'https://cve.mitre.org/cve/v5_00/',
+  type: 'object',
+  title: '',
+  description: 'a CVE entry',
+  definitions: {
+    cve_id: {
+      type: 'string',
+      pattern: '^CVE-[0-9]{4}-[0-9]{4,}$'
+    },
+    email_address: {
+      type: 'string',
+      minLength: 5,
+      pattern: '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,18})$'
+    },
+    data_type: {
+      type: 'string',
+      enum: [
+        'CVE'
+      ]
+    },
+    data_format: {
+      type: 'string',
+      enum: [
+        'MITRE'
+      ]
+    },
+    data_version: {
+      type: 'string',
+      enum: [
+        '5.0'
+      ]
+    },
+    CVE_data_meta_reject: {
+      type: 'object',
+      description: 'This is meta data about the CVE ID such as the CVE ID, who requested it, who assigned it, when it was requested, when it was assigned, the current state (PUBLIC, REJECT, etc.) and so on.',
+      required: [
+        'ID',
+        'ASSIGNER',
+        'STATE'
+      ],
+      properties: {
+        ID: {
+          $ref: '#/definitions/cve_id'
+        },
+        ASSIGNER: {
+          $ref: '#/definitions/email_address',
+          description: 'the assigner of the CVE (email address)'
+        },
+        STATE: {
+          type: 'string',
+          description: 'State of CVE - PUBLIC, RESERVED, REJECT',
+          enum: [
+            'REJECT'
+          ]
+        }
+      }
+      // additionalProperties: false
+    },
+    descriptions: {
+      type: 'array',
+      description: 'multi-lingual description of the vulnerability',
+      minItems: 1,
+      uniqueItems: true,
+      items: {
+        type: 'object',
+        description: '',
+        properties: {
+          lang: {
+            type: 'string',
+            description: 'ISO 639-2 language code',
+            default: 'EN',
+            minLength: 1
+          },
+          value: {
+            type: 'string',
+            description: 'Description of the vulnerability. Eg., [PROBLEMTYPE] in [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] allows [ATTACKER] to [IMPACT] via [VECTOR]. OR [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] [ROOT CAUSE], which allows [ATTACKER] to [IMPACT] via [VECTOR].',
+            minLength: 1
+          }
+        },
+        required: [
+          'lang',
+          'value'
+        ],
+        additionalProperties: false
+      }
+    }
+  },
+  properties: {
+    data_type: { $ref: '#/definitions/data_type' },
+    data_format: { $ref: '#/definitions/data_format' },
+    data_version: { $ref: '#/definitions/data_version' },
+    CVE_data_meta: { $ref: '#/definitions/CVE_data_meta_reject' },
+    descriptions: { $ref: '#/definitions/descriptions' }
+  },
+  required: [
+    'data_type',
+    'data_format',
+    'data_version',
+    'CVE_data_meta'
+  ],
+  additionalProperties: false
+}
+
 module.exports = {
   cveSchema4,
   cveSchema5,
```

Comment:
To be clear, that's because we broke it out into the three below it, right? If that's just a clean up task, I'd like to leave it for later to make sure we fix the lost work/git problem right now.

Reply:
Yeah, that's just a clean/up task.